service/src/test/java/com/ge/predix/controller/test/AttributeConnectorControllerIT.java

```diff
@@ -133,6 +137,15 @@ public void testCreateResourceConnectorWithCachedIntervalBelowThreshold(final St
                 .andExpect(status().isUnprocessableEntity());
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testZoneDoesNotExist() throws Exception {
+        MockMvcContext getContext =
+                TEST_UTILS.createWACWithCustomGETRequestBuilder(this.wac, "test-zone",
+                        "/v1/connector/resource");
+        getContext.getMockMvc().perform(getContext.getBuilder()).andExpect(status().isNotFound());
```

Comment:
Verify that the JSON body contains the text you are expecting. Use test ```testCreateAndGetAndDeleteConnector``` as a guide.